packages/perseus/src/validation.types.ts

```diff
@@ -286,3 +287,60 @@ export type UserInputMap = {[widgetId: string]: UserInput | UserInputMap};
 export type UserInputArray = ReadonlyArray<
     UserInputArray | UserInput | null | undefined
 >;
+export interface ValidationDataTypes {
```

Comment:
This uses the same construct as the new `PerseusWidgetTypes` does. However, the fact that not all widgets have validation data concerns me because it means that this interface won't have an entry for all widgets. 🤔 I'm not sure how to tackle that yet.

Reply:
What are the potential issues with the interface not having an entry for all widgets?

Reply:
Today I'm not totally sure yet. At least, it would give us an inaccurate view of what the different types of validation data we support (we might have a type defined, but not in this "master list"). Ideally in future PRs we can derive some extra checks to ensure these interface type registries stay accurate. 🤞